### PR TITLE
Add `credential_types` field to payload.

### PR DESCRIPTION
**Pull request by PopcornPaws**
# Description
As mentioned in #2 , if the `credential_types` field is missing from the `request` payload, it defaults to `orb`, even if `VerificationLevel` is set to `Device`. Therefore, the user&mdash;who isn't verified by an Orb yet&mdash;is prompted by a "You need to be verified by Orb to proceed this" error message. See image below.
world-id

Since #2 seems to be abandoned, I thought I'll open this PR as I also need this issue to be fixed.

### CODE REVIEW

**Review by 0xPenryn — APPROVED** [review]

**Comment by PopcornPaws** [comment]
Thanks for approving this PR @0xPenryn but I can't merge it until the workflows are run successfully. They seem to also need approval to run in the first place. Could you also approve/trigger the workflows to run?

**Comment by 0xPenryn** [comment]
> Thanks for approving this PR @0xPenryn but I can't merge it until the workflows are run successfully. They seem to also need approval to run in the first place. Could you also approve/trigger the workflows to run?

Running the workflows now, pinging @m1guelpf to merge and publish!

**Review by m1guelpf — APPROVED** [review]

**Pull request merged**
